src/emulator/portUtils.ts

```diff
@@ -495,3 +495,13 @@ export function listenSpecsToString(specs: ListenSpec[]): string {
     })
     .join(",");
 }
+
+export function findProcessBlockingPort(port: number): number | undefined {
+  try {
+    const res = execSync(`lsof -t -i :${port} `, {});
```

Comment:
sanity checking that this is our best option to check port

Reply:
Hmmm, we should probably use https://github.com/marionebl/lsofi instead to make sure this works on windows too. Will swap it over